app/models/project.rb

```diff
@@ -21,6 +21,10 @@ class Project < ActiveRecord::Base
 
   validates :name, presence: true,
                    uniqueness: { case_sensitive: false }
+  validate do |project|
+    ClientBillableValidator.new(project).validate
+  end
```

Comment:
If you move this into `app/validators/client_billable_validator.rb` you can do: `validates_with ClientBillableValidator`